css-four-column-layout/styles.css

```diff
@@ -0,0 +1,61 @@
+@import url('https://fonts.googleapis.com/css2?family=Barlow:wght@400;800&display=swap');
```

Comment:
Avoid using `@import` in CSS, it has very poor performance and the community no longer recommends it.